app/models/campaign.rb

```diff
@@ -42,7 +43,7 @@ class Campaign < ActiveRecord::Base
   # Scopes #
   ##########
 
-  scope :active, -> { where('end > ?', Time.zone.now) }
+  scope :active, -> { where('end > ? OR (start IS NULL AND end IS NULL)', Time.zone.now) }
```

Comment:
So then why not just check for end IS NULL? that matches the use case here, and isn't coupled to whether or not we validate co-presence of start and end.